app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for human in people:
+        each_person = Person(human["name"], human["age"])
+        person_list.append(each_person)
```

Comment:
Consider using list comprehension to create the `person_list`. This is recommended in the checklist for creating a list of instances from a collection.